### DISCUSSION TITLE
wrong program tab sorry!

### DISCUSSION THREAD

**Discussion by nbollman** [RWKV/rwkv.cpp#140] [General]
GGML format is expired, most of the pointers in the library point to GGML formats have updated gguf files available...

**Comment by saharNooby** [comment]
Hi! `rwkv.cpp` uses a [custom file format](https://github.com/saharNooby/rwkv.cpp/blob/master/docs/FILE_FORMAT.md) that is inspired by, but not identical to `ggml` format previously used in `llama.cpp`.

For the moment, I don't see any significant reasons to move `rwkv.cpp` to `gguf`. It may change when I'll be implementing RWKV v5 tho -- but I don't know in advance.
